### PR TITLE
Upgrade Dependencies

### PR DESCRIPTION
**Pull request by adrw**
* Bump Misk to public `0.8.0`
* Remove Square URLs
* Move protos to respective clients
* Use Wire Gradle Plugin

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -31,7 +32,8 @@ jobs:
   java:
     machine:
       image: circleci/classic:latest
-    parallelism: 2
+    parallelism: 1
+    working_directory: ~/barber
```
Comment (shellderp) [inline .circleci/config.yml]: s/barber/backfila/

File: dependencies.gradle [hunk 2]
```diff
@@ -23,20 +23,20 @@ ext.dep = [
...
   "mavenPublishGradlePlugin": "com.vanniktech:gradle-maven-publish-plugin:0.8.0",
   "metricsCore": "io.dropwizard.metrics:metrics-core:4.0.1",
   "metricsParent": "io.dropwizard.metrics:metrics-parent:4.0.1",
-  "misk": "com.squareup.misk:misk:2019.09.23-1569250868-5144703c",
-  "miskHibernate": "com.squareup.misk:misk-hibernate:2019.09.23-1569250868-5144703c",
-  "miskHibernateTesting": "com.squareup.misk:misk-hibernate-testing:2019.09.23-1569250868-5144703c",
-  "miskSlack": "com.squareup.misk:misk-slack:2019.09.23-1569250868-5144703c",
-  "miskTesting": "com.squareup.misk:misk-testing:2019.09.23-1569250868-5144703c",
-  "miskZipkin": "com.squareup.misk:misk-zipkin:2019.09.23-1569250868-5144703c",
+  "misk": "com.squareup.misk:misk:0.8.0",
```
Comment (shellderp) [inline dependencies.gradle]: new misk woo!

Comment (adrw) [reply dependencies.gradle]: I do it for the fans

File: .circleci/config.yml (hunk 1 above)
Comment (adrw) [reply .circleci/config.yml]: Fixed in a follow up

**Comment by adrw** [comment]
@shellderp Can you look at the java build failure WRT `invalid default value for 'lease_expires_at'`?

**Comment by shellderp** [comment]
> @shellderp Can you look at the java build failure WRT `invalid default value for 'lease_expires_at'`? Not sure whether there is some part of the code or CircleCI MySql setup I'm missing that has this working locally but not in CI.
> 
> Alternatively, could land this PR since it works locally and fix CircleCI in a follow up?

weird I'll take a look tonight. Seems fine to merge since this is a migration already applied in production..

**Comment by adrw** [comment]
I'll land this to unblock some other work I'm doing on `cash-backfila` and follow up with a fix once I understand Kochiku MySql settings better. Filed an issue here https://github.com/cashapp/backfila/issues/23

**Pull request merged**